es/criteria/require-review.md

```diff
@@ -20,26 +20,19 @@ order: 7
 
 * Aumenta la calidad de la codebase.
 * Reduce los riesgos de seguridad y los riesgos operativos.
-* Crea una cultura de hacer que cada contribución sea grande.
-* Atrapa los errores más obvios que puedan ocurrir.
+* Crea una cultura de hacer que cada contribución sea de alta calidad.
+* Detecta los errores más obvios que puedan ocurrir.
 * Da a los contribuyentes la seguridad de que sus contribuciones sólo se aceptan si realmente añaden valor.
 * Asegura a los contribuyentes un tiempo garantizado para la retroalimentación o la mejora colaborativa.
 
-## What this does not do
-
-* Guarantee the right solution to a problem.
-* Mean that reviewers are liable.
-* Absolve a contributor from writing documentation and tests.
-* Provide you with the right reviewers.
-
 ## Qué no hace
 
-* Garantizar la solución correcta de un problema.
-* Implicar que los revisores son responsables.
-* Absolver a un colaborador de escribir la documentación y los tests o pruebas.
-* Proporcionar los revisores adecuados.
+* Garantizar la solución adecuada a un problema dado.
+* Asegurar que los revisores son confiables.
+* Libera al contribuyente de tener que escribir documentación y pruebas.
+* Proveer los revisores adecuados.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"